### ISSUE TITLE
Why fixed point instead of float?

### ISSUE THREAD

**Issue by obsoleszenz** [Holzhaus/vinylla#16]
Is there any reason to use fixed point over float? I thought it was an optimization of the past :)

**Comment by Holzhaus** [comment]
Sorry, I'm lacking context here. Where is fixed point arithmetic used?

**Comment by obsoleszenz** [comment]
Maybe i'm just assuming it, but isn't this fixed point arithmetic?

https://github.com/Holzhaus/vinylla/blob/main/src/util.rs#L68

Also here the sample is a i32 ? I would have expected an f32 ?
https://github.com/Holzhaus/vinylla/blob/659fe1e9a9f50cab0b59b8ed7daaeccdf963ff47/src/timecode.rs#L72

**Comment by Holzhaus** [comment]
> Maybe i'm just assuming it, but isn't this fixed point arithmetic?
> 
> https://github.com/Holzhaus/vinylla/blob/main/src/util.rs#L68

No, that's just integer math. Not using fixed point arithmeric anywhere IIRC. 

> Also here the sample is a i32 ? I would have expected an f32 ?
> https://github.com/Holzhaus/vinylla/blob/659fe1e9a9f50cab0b59b8ed7daaeccdf963ff47/src/timecode.rs#L72

Originally the samples are an i16, because 16 bit audio should suffice for our purposes. The Serato timecode audio CDs are also 16 bit anyway.

https://github.com/Holzhaus/vinylla/blob/main/src%2Ftimecode.rs#L129-L130

Using integers here because it's faster and I don't see an advantage in using floating point math for this use case.
